app/main.py

```diff
@@ -1 +1,35 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self,
```

Comment:
Use [this](https://stackoverflow.com/a/41591509) styling for parameters with typings